_layouts/tag_page.html

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
1. **Tag index** is a page with all the tag results = https://federalist.18f.gov/preview/18F/beta.18f.gov/add-blog/tags/agency-work/

2. **Tag page** is a list of all the tags = https://federalist.18f.gov/preview/18F/beta.18f.gov/add-blog/tags/

Maybe those titles don't make sense. Maybe it should be tag results for #1 and tag index for #2?